<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>양/불 판정 분석 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#000000;&quot;&gt;양/불 판정 분석 프로그램&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dataframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;DataFrame&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_dataframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>731</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_datainfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Data Info&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sample">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;샘플 수 :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_sample">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_good">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;양품개수 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_good">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_bad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;불량개수 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_bad">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>104</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_datachart">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Data Chart&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_learning">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>580</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>학습화면으로 가기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_analysis_1d">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>분석화면(1D)으로 가기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_analysis_2d">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>분석화면(2D)으로 가기</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_datachart"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_data_analysis">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>450</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>데이터 분석으로 가기</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_open_api"/>
    <addaction name="action_open_local"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="file_menu"/>
  </widget>
  <action name="action_open_local">
   <property name="text">
    <string>열기(Local)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>다른 이름으로 저장</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>끝내기</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_open_api">
   <property name="text">
    <string>열기(API)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
